modules/upgrade-central-40.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-from-40-43.adoc
+:_module-type: PROCEDURE
+[id="upgrade-central-40_{context}"]
+= Upgrading Central
+
+You can update Central to the latest version by downloading and deploying the updated images.
+
+.Prerequisites
+
+* If you deploy images from a private image registry, first push the new image into your private registry, and then replace your image registry for the commands in this section.
+* If you used Red Hat UBI-based images when you installed {product-title}, replace the image names for the commands in this section with the following UBI-based image names:
+** For Central, Sensor, and Compliance use `\stackrox.io/main-rhel`
```

Comment:
s/Compliance use/Compliance, use
Also just confirming that `stackrox` is still applicable in these URLs

Reply:
Yeah its still `stackrox`, they still have to switch names.